.github/actions/linux-deps/action.yml

```diff
@@ -9,11 +9,7 @@ runs:
       run: |
         sudo apt-get update
         sudo add-apt-repository ppa:ubuntu-toolchain-r/test
-        sudo apt-get install gcc-13 g++-13 -y
```

Comment:
I wasn't able to find a gcc-13 with riscv. 

I think the riscv on ubuntu-20.04 is gcc-9.

Reply:
Hi @fire, is it necessary to downgrade this to gcc-9?

Reply:
I think I was trying to upgrade to 24.04 which has gcc 13, but I messed up.

Reply:
The main issue with using Ubuntu 24.04 is the GLIBC versions are different and many folks on certain distributions like Kubuntu for example are unable to run the compiled shared library because it uses a newer GLIBC version.

Reply:
The issue is trying to get a gcc-13 riscv cross compiler, not sure how to find one that isn't like 24.04.

Reply:
I can try it.

Reply:
@Naros as you requested. https://github.com/V-Sekai-fire/godot-orchestrator/actions/runs/9923424781/job/27413554767

Reply:
Feel free to edit my branch.